instruments/thorlabs/thorlabsapt.py

```diff
@@ -52,7 +54,17 @@ def enabled(self):
             Gets/sets the enabled status for the specified APT channel
 
             :type: `bool`
+
+            :raises TypeError: If controller is not supported
             """
+            if self._apt.model_number[0:3] == 'KIM':
+                raise TypeError("For KIM controllers, use the "
+                                "`enabled_single` function to enable "
+                                "one axis. For KIM101 controllers, "
+                                "multiple axes can be enabled using "
+                                "the `enabled_multi` function from the "
+                                "controller level.")
```

Comment:
According to the APT manual, the KIM101 should react to this command. However, mine does not whatsoever. In the new messaging system that is implemented, they have now commands to enable axes. I did not want to re-implement `enabled` because it might work with a KIM001 or a TIM101 controller.